masterlist.yaml

```diff
@@ -6690,6 +6690,16 @@ plugins:
       - crc: 0xFFA02D0D
         util: 'TES4Edit v4.0.3h (Hotfix 1)'
 
+  - name: 'Maskar''s Oblivion Overhaul.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/42780/'
+        name: 'Maskar''s Oblivion Overhaul'
+    group: *lateChangesGroup
+    after: [ 'Bashed Patch, 0.esp' ]  # The default filename for a Bashed Patch.
```

Comment:
This load after rule could be removed, the group already puts it after the bashed patch.
```
    after: [ 'Bashed Patch, 0.esp' ]  # The default filename for a Bashed Patch.
```